charts/descheduler/templates/configmap.yaml

```
@@ -1,3 +1,4 @@
+{{- if not  .Values.existingPolicyConfigMap }}
```

couldn't we just use the existing `deschedulerPolicy`?

```suggestion
{{- if .Values.deschedulerPolicy }}
```

---

Hi Amir,
Nice catch. I have implemented as recommended.  I also added a few comments on values.yaml to explain the new functionality